docs/deployment-httpd.md

```diff
@@ -6,9 +6,9 @@ description: Deploy your MyST site on Apache httpd
 
 Once you have built a static version of your MyST project via `myst build --html`, it can be served on an [Apache web server (httpd)](https://httpd.apache.org) instance.
 
-For example, let's assume your web server's base URL is `http://www.example.com`, it uses `/var/www/html` as its `DocumentRoot`, and that your MyST project's `_build` directory is `/var/www/html/my_MyST_project/_build`.
+For example, let's assume your web server's base URL is `http://www.example.com`, it uses `/var/www/html` as its `DocumentRoot`. You should copy the contents of your `_build/html/` directory into the `/var/www/html/` directory.
```

Comment:
With this rewording, the difference between root and non-root site disappears. I think the root version can be inferred from the other. Consider something like:

```suggestion
For example, let's assume your web server's base URL is `http://www.example.com`, it uses `/var/www/html` as its `DocumentRoot`. If you copy the `_build/html/` directory to `/var/www/html/my-project`, then the URL would be: `http://www.example.com/my-project`:

cp -r _build/html /var/www/html/my-project
```

(Minor suggestion, not a big deal.)